ballista/executor/src/executor_process.rs

```diff
@@ -181,11 +182,25 @@ pub async fn start_executor_process(opt: ExecutorProcessConfig) -> Result<()> {
         BallistaError::Internal("Failed to init Executor RuntimeEnv".to_owned())
     })?);
 
+    let mut service_handlers: FuturesUnordered<JoinHandle<Result<(), BallistaError>>> =
+        FuturesUnordered::new();
+
     let metrics_collector = Arc::new(LoggingMetricsCollector::default());
 
+    let (replicator_send, replicator_recv) = mpsc::channel::<replicator::Command>(10);
+    let replication_url =
+        ObjectStoreUrl::parse("authority").expect("Invalid default catalog url");
+
+    service_handlers.push(tokio::spawn(replicator::start_replication(
+        "",
+        runtime.object_store(replication_url)?,
+        replicator_recv,
```

Comment:
I probably need to clone here, to leave one sender unused to keep the channel alive for a whole exec lifetime